### ISSUE TITLE
Move inline editor/textarea editor widgets, and etc; to handlebars template

### ISSUE THREAD

**Issue by taboca** [taboca/s5s#327]
Which is probably moving into _slides_editor_service.html

**Comment by taboca** [comment]
Review: 

* slides_editor_tabs
* Slides_editor_tabs_editor 
* slides_editor_Tabs_images
 
````

<div id='post-item-editor-{{post.key}}' class='editingPostItem'>


		Suas ideias em {{post.date}}


		<div class='updatespanel'>
		</div>

		<div id='post-item-editor-toolbar-{{post.key}}' class='editingPostItemToolbarEditor'>
			
			<a class="btn btn-small btn-warning _editar" href="javascript:" onclick="postEdit('{{post.
			key}}');return false">
				Editar
			</a>
	
			<a class="btn btn-small btn-danger _salvar" style="display:none" href="javascript:" onclick="postSave('{{post.key}}');return false" >
				Salvar
			</a>

			<a class="btn btn-small btn-warning _anexar"  href="javascript:" onclick="postImage('{{post.key}}','{{articlekey}}');return false">
				Anexar
			</a>

			<select id='post-item-editor-kind-{{post.key}}'  >
			<option value='medium-box'>Medium Box</option>
			<option value='quote-box'>Quote</option>
			<option value='text-center'>Title</option>
			</select>

		</div>


		<div id='post-item-editor-images-{{post.key}}' >

			<div class='backpanel'>
			</div>

			{% if post.images %}
			<div class='imagesCollection' style='display:none'>
			{% for image in post.images %}
			       <a href='javascript:' style='float:right;margin:.3em;' onclick='setBackground("{{image.key()}}","{{article.key()}}");return false'>Image</a>
			{% endfor %} 
			</div>
			{% endif %}

			<!-- CAUTION DISPLAY NONE --> 

			<div class='semanticAuthor' style='display:none'>
			Por <a class='assertionAuthor' href='{{post.author}}'>{{post.author}}</a>
			em
			<span class='assertionDate'>{{post.date.strftime('%Y-%m-%d')}}</span>
			</div>

		<!-- CAUTION DISPLAY NONE --> 

		</div>

		<div class='editorPanel' data-id='{{post.key}}'>  

			<textarea id="editor-post-item-{{post.key}}" data-id="{{post.key}}" class="editorInlineForPosts" rows="3" >{{post.contentPlain|safe}}</textarea>

		</div>

</div>

````

**Comment by taboca** [comment]
Fixed by work in #328 and #330
